experiment/clustersecretbackup/README.md

```diff
@@ -0,0 +1,20 @@
+# clustersecretbackup
+
+Cluster secret backup is a tool backing up secrets in a cluster in Google Secret
+Manager.
+
+## Prerequisite
+
+- Set `GOOGLE_APPLICATION_CREDENTIALS`
```

Comment:
nit: Isn't this only necessary if using a service account key file?

Reply:
Yeah, I have added another option to authenticate